cmd/cortex/main.go

```diff
@@ -82,6 +83,13 @@ func main() {
 			}
 			os.Exit(1)
 		}
+	} else if len(args) == 0 {
+
+		fmt.Fprintf(os.Stderr, "Error loading config ,config.yaml does not exist, please set config.file for custom config path .for example:-config.file=./docs/configuration/single-process-config-blocks-local.yaml")
```

Comment:
No arguments is an error. That I can agree. But `config.yaml` is not the default. The default is empty

How about something like the following?
```suggestion
		fmt.Fprintf(os.Stderr, "please set configuration file. For example: -config.file=./docs/configuration/single-process-config-blocks-local.yaml")
```

Reply:
thanks